src/Lean/Meta/Tactic/TryThis.lean

```diff
@@ -183,10 +97,15 @@ The parameters are:
   suggestion does not have a custom `toCodeActionTitle?`. If not provided, `"Try this: "` is used.
 -/
 def addSuggestion (ref : Syntax) (s : Suggestion) (origSpan? : Option Syntax := none)
-    (header : String := "Try this: ") (codeActionPrefix? : Option String := none) : MetaM Unit := do
-  logInfoAt ref m!"{header}{s}"
-  addSuggestionCore ref #[s] header (isInline := true) origSpan?
-    (codeActionPrefix? := codeActionPrefix?)
+    (header : String := "Try this: ") (codeActionPrefix? : Option String := none) :
+    MetaM Unit := do
+  let hintSuggestion := {
+    span? := origSpan?
+    diffGranularity := .none
```

Comment:
Should we give an additional optional argument `diffGranularity := .none` to `addSuggestion`(`s`)? I can certainly imagine wanting to communicate "not that; this instead" using a diff when using `addSuggestion` in a linter (which is now useful! It at least adds a functional widget, but still fails to add the code action).